airflow/www/templates/airflow/confirm.html

```diff
@@ -28,10 +28,14 @@ <h2>Wait a minute.</h2>
       <pre><code>{{ details }}</code></pre>
     {% endif %}
   </div>
-  <form method="POST">
+  {% if endpoint %}
+    <form method="POST" action="{{ endpoint }}">
+  {% else %}
+    <form method="POST">
+  {% endif %}
```

Comment:
What are the possible values of `endpoint`? If it's always str (not e.g. `None`) then this is not needed; `action=""` is a valid value and does the right thing.